docs/production-deployment/cloud/billing-and-cost.mdx

```diff
@@ -30,7 +30,7 @@ tags:
   - invoice history
 ---
 
-The Account Owner holds the ability to access and view detailed billing and cost information at any time, ensuring full transparency and control over financial aspects. If you need to assess your spending patterns, check invoice histories, or handle payment details, Temporal Cloud's Cost and Billing capabilities are able to provide the information you need.
+The Account Owner and Finance Admin hold the ability to access and view detailed billing and cost information at any time, ensuring full transparency and control over financial aspects. If you need to assess your spending patterns, check invoice histories, or handle payment details, Temporal Cloud's Cost and Billing capabilities are able to provide the information you need.
```

Comment:
Do you want to link to the billing and cost page? https://docs.temporal.io/cloud/billing-and-cost

Reply:
They didn't ask for this, but I think that would be a great addition. Adding.